.eslintrc.js

```diff
@@ -11,13 +11,14 @@ module.exports = {
     "array-callback-return": "off",
     "linebreak-style": "off",
     "no-console": 1,
+    "import/no-extraneous-dependencies": ["error", {"devDependencies": ["**/test-utils/**", '**/*test.js']}]
```

Comment:
what's this about?

Reply:
stops the squiggly lines under the nock and test cafe as lint thinks they should be in dependancies not dev dependancies

Reply:
ok 👍